modules/toggleSection.js

```diff
@@ -0,0 +1,16 @@
+const toggleSectionVisibility = (sectionId) => {
+  const listBookSection = document.getElementById('list-books-section');
+  const addBooksSection = document.getElementById('add-books-section');
+  const contactSection = document.getElementById('contact-section');
+  const sections = [listBookSection, addBooksSection, contactSection];
+
+  sections.forEach((section) => {
+    if (section.id === sectionId) {
+      section.style.display = 'block';
+    } else {
+      section.style.display = 'none';
+    }
+  });
+};
+
+export default toggleSectionVisibility;
```

Comment:
- [ ] Great job so far refactoring your code. Everything looks good so far however the navigation of your application isn't working as expected due to that I was unable to test the `add` and `remove` functionalities of your app. 

   - [ ]  Currently, when a user navigates to `Add new` and `Contact` a `404` error is returned to the UI. Kindly make sure that your app navigation is working currently. This will help improve the user experience. See below for more details:
<img width="1060" alt="Screenshot 2023-08-14 at 21 30 46" src="https://github.com/egichuhis/Awesome-Books-ES6/assets/107192076/e6a8b971-beff-48ea-ae8e-86db4fbe799f">